spec/features/course/achievement_management_spec.rb

```diff
@@ -69,33 +69,60 @@
       end
 
       scenario 'I can create an achievement condition' do
+
+        # No achievements
+        Course::Condition::Achievement.destroy_all
+        Course::Achievement.destroy_all
         achievement = create(:course_achievement, course: course)
         visit edit_course_achievement_path(course, achievement)
-        condition_achievement = create(:course_achievement, course: course)
-        expect(page).to have_selector(:link_or_button,
-                                      I18n.t('course.condition.achievements.new.header'))
+        expect(page).not_to have_selector(:link,
+                                          I18n.t('course.condition.achievements.new.header'))
+
+        new_course, achievement, achievement_condition, other_achievement =
```

Comment:
Useless assignment to variable - `achievement_condition`. Use `_` or `_achievement_condition` as a variable name to indicate that it won't be used.